gcp-native-ts-k8s-ruby-on-rails-postgresql/infra/config.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import { Config } from "@pulumi/pulumi";
+
+const config = new Config();
+
+const gcpConfig = new Config("gcp");
```

Comment:
Now I see why you used `gcp` and not `gcp-native`. I guess it's fine for now to avoid duplication - hopefully we can get rid of `gcp` soon.

Reply:
I switched to use google-native config instead and just added the project reference to the gcp DB user for now.